src/replicache.ts

```diff
@@ -694,6 +694,10 @@ export default class Replicache implements ReadTransaction {
 
     return {result, ref};
   }
+
+  async setLogLevel(level: 'debug' | 'info' | 'error'): Promise<void> {
```

Comment:
Can we call this `setWasmLogLevel` or something? Along with following comment:

```
Sets the log level the internal Replicache Wasm binary will use.

By default, the Wasm library logs at info level, and the JavaScript wrapper code logs at whatever level is set in the web inspector. Unfortunately there is no way to know when the log level in the inspector changes and forward that information to Wasm.

So if you want to enable verbose logging for Replicache you must set the inspector to log at `debug` level and also call `setLogLevel('debug')` on a `Replicache` instance.
```

Reply:
Simpler:

```
// Enables extra logging from the Replicache wasm module.
//
// This is disabled by default for performance reasons, but it can be useful to enable when
// debugging something.
void enableVerboseWasmLogging()
```